platform/mds/mds-web/src/main/java/org/motechproject/mds/web/util/Constants.java

```diff
@@ -0,0 +1,10 @@
+package org.motechproject.mds.web.util;
+
+public final class Constants {
```

Comment:
1) javadoc
2) do we need these constants? don't we already have these defined in mds?